raidprotect/src/event/message/handle.rs

```diff
@@ -14,5 +21,41 @@ pub async fn handle_message(message: Message, state: Arc<ClusterState>) {
     let parsed = parse_message(&message);
     state.redis().set(&parsed).await.ok();
 
+    if is_an_old_command(&message) {
```

Comment:
This should be handled in a separate task to avoid blocking other (future) message processing modules.

```rs
if is_old_command(&message.content) {
     let message = message.clone();
     let state = state.clone();

     tokio::spawn(warn_old_command(message, state));
}
```